bite/BiteManager.h

```diff
@@ -36,16 +38,41 @@ class BiteManager {
 
 public:
     explicit BiteManager(Schain &_schain);
-
     ~BiteManager();
 
+    // =============== Transaction Parsing Calls =============== //
+
+    // Runs through all transactions in the proposal and tries to parse all BITE transactions.
+    // This includes both:
+    // 1) BITE1 transactions - with both 'to' and 'data' fields encrypted
+    // 2) BITE2 transactions - with only function arguments encrypted, placed in the data field.
+    //
+    // Unparsable transactions will be added to failedTransactions.
+    // Transactions starting from the magic number but with incorrect format will be added
+    // to failedTransactions.
     static void parseBITETransactions(ptr<BlockProposal> _proposal);
 
-    // this will return a map of failed transactions
-    // if none of the transactions fails, the  proposal is set with decryption shares
+    // Tries to match the TO field to BITE1 magic number. If it matches - tries to parse the BITE1 data field.
+    // If parsing is successful - returns BiteCiphertext object, else returns 'nullopt'
+    static ptr<BiteCiphertext> tryGetEncryptedRegularTxFields(
```

Comment:
does it make sense to use `tryGetEncryptedBITE1TxFields`? `ENCRYPTED REGULAR TX` may not be very clear